frontend/src/pages/CocktailRecipePage.tsx

```diff
@@ -1,15 +1,21 @@
 import {useParams} from "react-router-dom";
 import CocktailRecipe from "../compononents/CocktailRecipe";
 import useCocktailRecipe from "../hooks/useCocktailRecipe";
-export default function CocktailRecipePage() {
+import {Favourite} from "../model/Favourite";
 
-    const params = useParams()
+type CocktailRecipeProps = {
+    addFavourite: (favourite: Favourite) => void
+}
+
+export default function CocktailRecipePage( { addFavourite} : CocktailRecipeProps ) {
+
+        const params = useParams()
     const id = params.id
     const {cocktailRecipe} = useCocktailRecipe(id)
 
     return (
         <div>
-            {cocktailRecipe && <CocktailRecipe cocktail={cocktailRecipe}/>}
+            {cocktailRecipe && <CocktailRecipe cocktail={cocktailRecipe}addFavourite={addFavourite}/>}
```

Comment:
Auch hier würde ich noch ein Leerzeichen empfehlen:

```suggestion
            {cocktailRecipe && <CocktailRecipe cocktail={cocktailRecipe} addFavourite={addFavourite}/>}
```